src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,24 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String res = "";
```

Comment:
read [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) attentively
Use StringBuilder instead of String or StringBuffer.

Reply:
ой, сорян)
спасибо!

Reply:
done